data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py

```diff
@@ -0,0 +1,129 @@
+"""
+The de-id scripts removes all fields in the person table except for the person_id and the birthdate_time field.
+Before CDR handoff to the Workbench team, we need to repopulate the following fields with demographic information
+from the observation table.
+
+These are the following fields in the person table will be repopulated:
+
+gender_concept_id
+year_of_birth
+month_of_birth
+day_of_birth
+race_concept_id
+ethnicity_concept_id
+gender_source_value
+gender_source_concept_id
+race_source_value
+race_source_concept_id
+ethnicity_source_value
+ethnicity_source_concept_id
+"""
+import logging
+
+import constants.bq_utils as bq_consts
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
I think we can remove this line and the `import logging` line, because nothing is logged from this module.

Reply:
^^